src/main/scala/nl/knaw/dans/api/sword2/Authentication.scala

```diff
@@ -25,7 +25,7 @@ object Authentication {
       throw new SwordError("http://purl.org/net/sword/error/MediationNotAllowed")
     }
     // temporary short-circuit
-    if (!(auth.getUsername == SwordProps.get("user")) || !(auth.getPassword == SwordProps.get("password"))) {
+    if (!(auth.getUsername == SwordProps("user")) || !(auth.getPassword == SwordProps("password"))) {
```

Comment:
Yes, this is temporary since it only works for one user (configured in the properties). The LDAP implementation will make this obsolete.